client/src/components/Home/HomeGridVideoBar/HomeGridVideoBar.tsx

```diff
@@ -100,20 +100,31 @@ const HomeGridVideoBar: FC<HomeGridVideoBarProps> = ({ className }) => {
       title={t('learnHowToUseOctant')}
       titleSuffix={
         <>
-          {!isMobile && <NavigationArrows {...navigationArrowsProps} />}
+          {!isMobile && (
+            <NavigationArrows
+              {...navigationArrowsProps}
+              dataTest="HomeGridVideoBar__NavigationArrows"
```

Comment:
If one is `--mobile`, the other should be...
```suggestion
              dataTest="HomeGridVideoBar__NavigationArrows--desktop"
```